### ISSUE TITLE
Process list CPU usage is wildly wrong

### ISSUE THREAD

**Issue by LouiseTGUK** [oguzhaninan/Stacer#438]
The cpu usage for some processes does not change or updates very slowly. 

if i open multiple tabs in browser cpu goes up as expected, but when i close them again CPU slowly drops, but the dash is showing the total CPU is much lower than the individual app.

When launching stacer it shows up to 70% CPU usage for stacer, and that figure very slowly reduces over a minute or two while dash is showing a typical idle usage of about 2 to 5%

htop shows dash CPU is correct as far as I can tell, but processes are completely wrong

Manjaro KDE-Plasma 5.20.5
intel core i7 @ 3.4ghz

![Screenshot_20210224_101900](https://user-images.githubusercontent.com/79573434/108986217-cc975900-7689-11eb-8d3c-2cfaefde3768.jpg)
![Screenshot_20210224_101911](https://user-images.githubusercontent.com/79573434/108986218-cd2fef80-7689-11eb-95b6-292690d16e81.jpg)

**Comment by gggirlgeek** [comment]
I agree. My Stacer process always reads around 17.3% without changing much but the dashboard says 4-5%. The built in Gnome System Monitor says the total cpu% is 4-5% and it also says the Stacer process is running at 17-19%! They agree.

I am running Stacer as sudo -E (so it uses my configuration with root privileges) but Gnome System Monitor is running as regular user. So user privileges don't seem to make a difference. 

It seems that only 1 cpu core is being measured for the cpu total in both cases. The Resources tab on Gnome System Monitor often shows other cpu cores spiking at 25% but that is not reflected in its' system tray. I wonder if Linux is able to add the cpu frequencies together?
